.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Can we please remove this and have
```
echo "use flake ..#dens-validation" > dens-validation/.envrc
```
instead?

E.g. we did this in lambda buffers with:
- https://github.com/mlabs-haskell/lambda-buffers/blob/main/lambda-buffers-codegen/.envrc
- https://github.com/mlabs-haskell/lambda-buffers/blob/a8b48775e6d80c9ad9fee0861b83fd844aeb8e76/lambda-buffers-codegen/build.nix#L25